### DISCUSSION TITLE
Creality K1

### DISCUSSION THREAD

**Discussion by Jeff0100** [OrcaSlicer/OrcaSlicer#1285] [Ideas]
I need a printer profile for the creality K1 🙏

**Comment by fr3ak2402** [comment]
I'm already working on it and it should be included in the next version.

**Reply by JimN89** [reply]
Around what time is the next version coming out?

**Reply by fr3ak2402** [reply]
I have already completed the profiles and they are in my repository. But these should be used with caution as they are already adapted to the large change in profiles.

If you are currently only using a Bambulab printer or no printer at all with the OrcaSlicer, you can simply download my repository and exchange the profiles folder on the resources folder with the profiles folder on your installation folder. Then you can already use the K1 profiles.

But if you use other 3rd party printers with the OrcaSlicer, then you will run into problems...

**Reply by Crazy-Guy** [reply]
I can't see any K1 profiles in any of the branches 🧐 Can you provide more information?

**Reply by TeeJS** [reply]
I believe he's talking about GalaxySlicer:  https://github.com/fr3ak2402/GalaxySlicer/releases

**Reply by Crazy-Guy** [reply]
✅ Thanks. That helped 😃

**Reply by Potas71** [reply]
I downloaded the new version "OrcaSlicer_V1.6.4-beta2_Win64", downloaded the profile repository and exchange the profiles folder, but did not worked, I cannot still see the printers. Used to work on version V1.6.3

**Comment by chemic** [comment]
Will there be release sure with K1 profile in it? Thanks a lot for pushing the community ♳

**Comment by TV-Technics** [comment]
I need a profile for the K1 max. Will it be also included in the next update?

**Reply by TV-Technics** [reply]
For orca slicer I mean

**Reply by andy-81** [reply]
Hi TV-Technics,

@fr3ak2402 has done a great job of the profiles under his GalaxySlicer. I only use Orca at present with the K1 Max and you can just copy the profiles into Orca and use them. I copied just the Creality folder under Profiles along with the Creality.json file in the Profiles folder and this appeared to work.

I haven't had the printer as yet so can not comment on the full outcome, but I was able to add the printer (all 3 nozzle sizes) and  filaments and was able to then select and slice using them. I did find the estimated time to print was slightly slower than when I used the Creality slicer however it may be more accurate in Orca and I am happy for the print to be slightly slower if I get a better print (I watched a number of videos on the difference slicers make and Orca seemed to do a good job)

Use the same link above provided for the K1 and follow the process mentioned by fr3ak2402

**Reply by TV-Technics** [reply]
I've imported a K1 max profile but orca says the printer is offline. How can I get my slices sended to the printer? (Sorry, I am new to this)

**Comment by andy-81** [comment]
Has anyone worked out what G-Code would be needed to add a filament change on the K1 and K1 Max?

I currently run Marlin on a modified Ender3 and use the M600 GCode to initiate the pause, unload and purge and was wondering if anyone knew what code should be used to do the same on the K1.

**Reply by Charlyfreak** [reply]
does anyone have a profile of the K1 Max for the Orca Slicer? I only have the K1 profile

**Reply by TV-Technics** [reply]
https://www.printables.com/group/creality-k1k1-max-group-b5KX4vR

**Comment by 3dprintingmatthew** [comment]
Hi everyone,

I have a profile which will work with prusaslicer/ superslicer/ orcaslicer. The link to download it can be found here:

https://3dprintingspace.com/t/creality-k1-prusa-slicer-profile/6861/6
